### PR TITLE
new component for assay run data 

### PR DESCRIPTION
**Pull request by labkey-chrisj**
#### Rationale
Entity Insert Panel seems to use different text for the mode-select buttons in different contexts
In Biologics, the assay upload panel looks like this:

image
for sample upload, like this:
image

Currently, the showGrid() method assumes the words 'from Grid' will be in the button to show the grid; in the assay case the text includes 'into Grid'- this change makes the helper work in both cases

#### Related Pull Requests
cross-folder assay automation

#### Changes
Just changes the matching text for the show grid button

### CODE REVIEW

**Review by labkey-tchad — APPROVED** [review]

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
Correction. It looks like the assay upload is a different component ([RunDataPanel.tsx](https://github.com/LabKey/labkey-ui-components/blob/develop/packages/components/src/internal/components/assay/RunDataPanel.tsx)).

**Comment by labkey-chrisj** [comment]
Is this a call for a separate test component?  I can do that

File: src/org/labkey/test/components/ui/entities/EntityInsertPanel.java [hunk 1]
```diff
@@ -304,7 +304,7 @@ public EntityInsertPanel showGrid()
     {
         if (!isGridVisible())
         {
-            modeSelectListItem("from Grid")
+            modeSelectListItem(" Grid")
```
Comment (labkey-danield) [inline src/org/labkey/test/components/ui/entities/EntityInsertPanel.java]: `modeSelectListItem("from Grid")` is used in several places in this file, why not be consistent and change them all?

**Comment by labkey-tchad** [comment]
> Is this a call for a separate test component? I can do that

If it's not too much trouble but don't feel like you need to blow up the scope of this story.

**Review by labkey-tchad — APPROVED** [review]
New component isn't a blocker. Maybe add a TODO if it would be too much trouble

File: src/org/labkey/test/components/ui/entities/EntityInsertPanel.java (hunk 1 above)
Comment (labkey-chrisj) [reply src/org/labkey/test/components/ui/entities/EntityInsertPanel.java]: I decided to not change EntityInsertPanel- I've restored the original text.
The new component (RunDataPanel) uses an enum to match the mode-button texts, hopefully that's not over-doing it

**Pull request merged**